source/network/packets/AddPlayerPacket.cpp

```diff
@@ -29,17 +27,13 @@ void AddPlayerPacket::write(RakNet::BitStream* bs)
 	bs->Write(m_guid);
 	bs->Write(m_name);
 	bs->Write(m_id);
-	bs->Write(m_x);
-	bs->Write(m_y);
-	bs->Write(m_z);
+	bs->Write(m_pos);
```

Comment:
I can understand the argument for clarity, but just to be clear, doing what you suggested would be the opposite of brevity, since it would make the code contents less brief by elongating them.